TACC/Core-Styles
# enhance: DSAPP-57 limit height of c-app-card desc

## Overview

Limit height of `c-app-card` desc, so [DSAPP-57](https://tacc-main.atlassian.net/browse/DSAPP-57) can be fixed.

## Related

- [DSAPP-57](https://tacc-main.atlassian.net/browse/DSAPP-57)
- required by https://github.com/DesignSafe-CI/portal/pull/1474

## Changes

- **changed** `.c-app-card` description to be truncated many lines

## Testing

1. Open http://localhost:3000/components/preview/c-app-card.
2. Change any description to have 3 or more lines of text.
3. Verify text is truncated (thus height is limited.)

## UI

| before | after |
| - | - |
| <img width="920" alt="before" src="https://github.com/user-attachments/assets/29737c5b-f17e-4dea-8d06-bdbc3020c39d"> | <img width="920" alt="after" src="https://github.com/user-attachments/assets/93f51a7f-cfb7-4465-809b-113aceffc444"> |
